### ISSUE TITLE
Move user_id out of cbsd resource

### ISSUE THREAD

**Issue by jkmar** [magma/domain-proxy#222]

**Comment by xbend** [comment]
Because of priority shifts, we will postpone realising this task in favor of:
- preparing work plan proposals and analysis of backend implemention options
- preparing topic discussions for community representative regarding orc8r services design and APIs
- unblocking UI team

**Comment by xbend** [comment]
Rephrasing.
CBSD user_id should be still part of cbsd entity, but if not set, Domain Proxy should have some global config value, which should be used as user_id when cbsd doesnt have override.
